### DISCUSSION TITLE
Recommended dev env setup?

### DISCUSSION THREAD

**Discussion by Alteras1** [JiLiZART/BBob#245] [Q&A]
Hello!

I'm looking to more actively contribute to BBob, and would like to know what is the recommended setup for developing within and making code changes to BBob. I've been making do with just creating unit tests and checking the resulting behavior at the surface level. I imagine there's probably a better way to develop in this monorepo, but I'm honestly at a lost on how to go about it.

**Comment by JiLiZART** [comment]
I think I can write some sort of CONTRIBUTION guide with instructions of dev setup
